tests/baselines/reference/tsxFragmentErrors.errors.txt

```diff
@@ -12,18 +12,16 @@ tests/cases/conformance/jsx/file.tsx(11,17): error TS1005: '</' expected.
     	}
     }
     declare var React: any;
-    
-    <>hi</div> // Error
-    ~~~~~~~~~~~~~~~~~~~
-          ~~~
-!!! error TS17015: Expected corresponding closing tag for JSX fragment.
-              ~~~~~~~~~
+                           
     
     
+    <>hi</div> // Error
+    ~~
+!!! error TS17014: JSX fragment has no corresponding closing tag.
+        ~~~~~~
+!!! error TS1005: 'SOMETHING CORRECT' expected.
     
     <>eof   // Error
-    ~~~~~~~~~~~~~~~~
-!!! error TS2657: JSX expressions must have one parent element.

```

Comment:
This error seems to get dropped because we're eagerly eating up tags after `<>` (whereas this test assumed that it would stop earlier, and therefore wrote two tests on two different lines).